card/lib/card/loader.rb

```diff
@@ -21,6 +21,40 @@ def load_mods
         load_set_patterns
         load_formats
         load_sets
+
+        if ENV['RAILS_ENV'] == "development"
+          update_machine_output_hack
+        end
+      end
+
+      def update_machine_output_hack
+        script = Card["*all+*script"]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.